File: apps/chat/libs/chat-domain/src/main/java/com/example/chat/domain/schedule/CronExpression.java
package com.example.chat.domain.schedule;

/**
 * 단순한 Cron 표현식 래퍼 (테스트용 최소 구현)
 */
public record CronExpression(String value) {
    public static CronExpression of(String value) {
        return new CronExpression(value);
    }

    public String getValue() { return value(); }
}

File: apps/chat/libs/chat-domain/src/main/kotlin/com/example/chat/domain/schedule/CronExpression.kt
package com.example.chat.domain.schedule

/**
 * 단순한 Cron 표현식 래퍼 (테스트용 최소 구현)
 */
class CronExpression(private val value: String) {
    fun getValue(): String = value

    companion object {
        @JvmStatic
        fun of(value: String): CronExpression = CronExpression(value)
    }
}
